remove cloud simulator doc pages (#1425)
Qiskit/documentation: docs/api/migration-guides/local-simulators.mdx

# Migrate from cloud simulators to local simulators

> In quantum computing, the choice between using simulators and quantum hardware is crucial for making progress in the field. While simulators are useful for testing and debugging, in this era of quantum utility, quantum development and industry advancement requires actual hardware. As part of the move to quantum utility, IBM Quantum&trade; will retire its cloud simulators on 15 May 2024. This guide explains in more detail why this is happening, and how to migrate from cloud-based simulators, such as `ibmq_qasm_simulator`, to local simulators.

Reviewer: Should this use past tense?

> In quantum computing, the choice between using simulators and quantum hardware is crucial for making progress in the field. While simulators are useful for testing and debugging, in this era of quantum utility, quantum development and industry advancement requires actual hardware. As part of the move to quantum utility, IBM Quantum&trade; will retire its cloud simulators on 15 May 2024. This guide explains in more detail why this is happening, and how to migrate from cloud-based simulators, such as `ibmq_qasm_simulator`, to local simulators.

Author: ```suggestion
In quantum computing, the choice between using simulators and quantum hardware is crucial for making progress in the field. While simulators are useful for testing and debugging, in this era of quantum utility, quantum development and industry advancement requires actual hardware. As part of the move to quantum utility, IBM Quantum&trade; cloud simulators were retired on 15 May 2024. This guide explains the retirement in more detail, and how to migrate from cloud-based simulators, such as `ibmq_qasm_simulator`, to local simulators.
```
